src/components/Xmp.react.js

```diff
@@ -23,7 +24,12 @@ const Xmp = (props) => {
 
```

Comment:
It would make sense to also omit `loading_state` from the props being passed down to the child

Reply:
Since the Loading component checks the `loading_state` prop, if we were to omit it, the Loading component would not work anymore, so I don't think we should omit it.